Makefile

```diff
@@ -37,13 +42,16 @@ test-w: ${NODE}
 build-module: src/*
 
 serve:
-	$(BABEL_NODE) server.js
+	open dist/index.html
```

Comment:
quick googling suggests `google-chrome` should work for you. i'll add the branching and you can let me know if it works.